bynder_sdk/util.py

```diff
@@ -1,3 +1,13 @@
+with open('VERSION') as fh:
+    __version__ = fh.read().strip()
+    fh.close()
```

Comment:
Unless you call `open()` with `closefd=False` it should automatically close: https://docs.python.org/3/library/functions.html#open

Reply:
Yep that's one of the main features of the `with` statement.